shared/src/data/constants/Addresses.ts

```diff
@@ -0,0 +1 @@
+export const UNISWAP_PERMIT2_ADDRESS = '0x000000000022D473030F116dDEE9F6B43aC78BA3';
```

Comment:
Now that we're using Typescript files for the ABIs, I think it'd be reasonable to put these sort of addresses in there. Especially when it's a universal address across all chains.

This would also make imports cleaner. e.g. `import {PERMIT2_ADDRESS, permit2ABI} from 'src/abis/Permit2.ts'`